app/src/main/java/org/schabi/newpipe/util/LinkHelper.java

```diff
@@ -0,0 +1,116 @@
+package org.schabi.newpipe.util;
+
+import android.content.Context;
+import android.text.SpannableStringBuilder;
+import android.text.method.LinkMovementMethod;
+import android.text.style.ClickableSpan;
+import android.text.style.URLSpan;
+import android.text.util.Linkify;
+import android.view.View;
+import android.widget.TextView;
+
+import androidx.core.text.HtmlCompat;
+
+import io.noties.markwon.Markwon;
+import io.noties.markwon.linkify.LinkifyPlugin;
+
+public final class LinkHelper {
```

Comment:
`LinkHelper` doesn't feel like the right name to me, though I don't have a better suggestion

Reply:
What do you think of something like LinkCreator or LinkTransformer?

Reply:
`TextLinkifier`, maybe?